sysctl not in PATH

Hi

https://github.com/philpep/testinfra/blob/99b9b6436ededf30496f47de53effda3a96a6cea/testinfra/modules/sysctl.py#L29

What if sysctl is not in the PATH? (typically /sbin/sysctl) 
Would you recommend to depend on another playbook or role to configure the user to change the PATH before being able to run this test?
Or to run this test with root privilege (which should have the correct PATH), while no root privileges are required?

Could I suggest that you try to find sysctl in /sbin if it is not in the PATH? (Actually you would just have to append /sbin at the end of the user's path when executing this command)

Today I'm getting this error:
    =================================== FAILURES ===================================
    _______________________ test_sysctls[ansible://instance] _______________________

    host = <testinfra.host.Host object at 0x7ff31ab701a8>

        def test_sysctls(host):
    >       assert host.sysctl('fs.file-max') == '6815744'

    tests/test_default.py:11:
    _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    self = <sysctl>, name = 'fs.file-max'

        def __call__(self, name):
    >       value = self.check_output("sysctl -n %s", name)
    E       AssertionError: Unexpected exit code 127 for CommandResult(command='sysctl -n fs.file-max', exit_status=127, stdout='', stderr='/bin/sh: sysctl: command not found')
    E       assert 127 == 0
    E        +  where 127 = CommandResult(command='sysctl -n fs.file-max', exit_status=127, stdout='', stderr='/bin/sh: sysctl: command not found').rc

    ../../../../ansible-virtualenv/lib/python3.6/site-packages/testinfra/modules/sysctl.py:29: AssertionError
    =========================== 1 failed in 2.39 seconds ===========================

--- a/testinfra/modules/sysctl.py
+++ b/testinfra/modules/sysctl.py
@@ -14,6 +14,7 @@
 from __future__ import unicode_literals
 
 from testinfra.modules.base import InstanceModule
+from testinfra.utils import cached_property
 
 
 class Sysctl(InstanceModule):
@@ -24,9 +25,12 @@ class Sysctl(InstanceModule):
     >>> host.sysctl("vm.dirty_ratio")
     20
     """
+    @cached_property
+    def _sysctl_command(self):
+        return self.find_command('sysctl')
 
     def __call__(self, name):
-        value = self.check_output("sysctl -n %s", name)
+        value = self.check_output("%s -n %s", self._sysctl_command, name)
         try:
             return int(value)
         except ValueError:

